### PR TITLE
CHANGELOG for 0.0.7

### PR DESCRIPTION
**Pull request by xorye**
Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: microprofile.ls/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # MicroProfile Language Server Changelog
 
+## 0.0.7 (April 30, 2020)
+
+### Enhancements
+
+ * Create Container Image snippets, update Quarkus datasource snippets. See [#310](https://github.com/redhat-developer/quarkus-ls/pull/310)
```
Comment (xorye) [inline microprofile.ls/CHANGELOG.md:7]: This PR is not merged yet

**Comment by fbricon** [comment]
At least https://github.com/redhat-developer/quarkus-ls/pull/309 is missing. Please check with the PRs that have been merged, that might not have been added to the milestone

**Comment by xorye** [comment]
Added

**Comment by xorye** [comment]
Changed to `Update Quarkus datasource snippet.`

**Pull request merged**